### PR TITLE
Cron + percent notifications

### PR DESCRIPTION
**Pull request by JasonMillward**
Gives the ability for a user to get regular notifications every X minutes.
https://github.com/thijsbekke/OctoPrint-Pushover/issues/25#issuecomment-456058714

### CODE REVIEW

File: octoprint_pushover/__init__.py [hunk 1]
```diff
@@ -152,6 +172,14 @@ def sent_m70(self, comm_instance, phase, cmd, cmd_type, gcode, *args, **kwargs):
 		:param kwargs: 
 		:return: 
 		"""
+		if gcode and gcode != "G1":
```
Comment (JasonMillward) [inline octoprint_pushover/__init__.py]: This is so we don't calculate the time difference every `G1` which is send multiple times a second.

**Comment by JasonMillward** [comment]
Should handle https://github.com/thijsbekke/OctoPrint-Pushover/issues/17 now too.

**Comment by thijsbekke** [comment]
Nice work! Unfortunately I can not merge this yet. There must be some kind of mechanism that this functionality does not use the default API key.  Otherwise that API key will reach his limit very quick. 

Very exited to merge this, and many people will enjoy it I guess. Maybe I can find some time tomorrow, too help.

**Comment by JasonMillward** [comment]
Perhaps a notification to the front end too?
```
self._plugin_manager.send_plugin_message(self._identifier, dict(type="popup", msg="Please use your own API key for this functionality"))
```

**Comment by JasonMillward** [comment]
image


Now the new functionality can't be used with the default API key.

**Pull request merged**